### PR TITLE
Projects page data visualizations

### PR DESCRIPTION
**Pull request by tehandyb**
The functionality and layout is done. Right now the stacked bar chart is still using dummy data, and the projects data is only partially imported into the app because there were data cleanliness issues

### CODE REVIEW

**Comment by colinnichols** [comment]
A few things,
 - looks like practice area wasnt translated into markdown correctly, it should be making a list of practice areas and they should be spelled the same as the practice areas on the site
 - the description wasn't converted correctly either, this should be below the markdown meta fields, see the markdown example I sent you in Slack
 - the regions look like they might still need to be cleaned up
 - I didn't see any publication markdown files, did we get enough data in the spreadsheet to create publications?

File: _layouts/project.html [hunk 1]
```diff
@@ -2,8 +2,16 @@
 layout: default
 ---
 
-{% if page['Practice Area'].size > 0 %}
-	{% assign header_title = page['Practice Area'] | first %}
+{% assign foundPracticeAreas = site.newEmptyArray %}
```
Comment (colinnichols) [inline _layouts/project.html:5]: @tehandyb Is this change needed since you updated how the project practice area lists are built?

**Comment by colinnichols** [comment]
Merging for demo purposes, data should still be updated for the projects and publications.

File: _layouts/project.html (hunk 1 above)
Comment (tehandyb) [reply _layouts/project.html:5]: Yeah this thing is still needed
